tools/code_coverage/package/util/setting.py

```diff
@@ -22,8 +22,8 @@
 
 # test type, DO NOT change the name, it should be consistent with [buck query --output-attribute] result
 class TestType(Enum):
```

Comment:
These need to inherit from Enum + str or StrEnum in the future I think

Reply:
As per [this changelog](https://github.com/python/mypy/blob/master/CHANGELOG.md#change-to-enum-membership-semantics), it seems like you only need to use typing.cast if the value isn't known.  Presumably it can figure out the type if you explicitly set it.  So I believe this is correct as is, but let me know if you think otherwise